### PR TITLE
Added Additional Attributes

### PR DESCRIPTION
**Pull request by mash2k3**
Added attributes to the zone switch such as:
Next start time and next start program

Also added 3 attributes while switch is active:
Current watering program, runtime and station

### CODE REVIEW

File: custom_components/bhyve/switch.py [hunk 1]
```diff
@@ -535,13 +580,17 @@ def _on_ws_data(self, data):
             event == EVENT_CHANGE_MODE and data.get("mode") in ("off", "auto")
         ):
             self._is_on = False
-            self._set_watering_started(None)
+            self._set_watering_started(None,None,None,None)
```
Comment (sebr) [inline custom_components/bhyve/switch.py:583]: nit: fix formatting

**Review by sebr — CHANGES_REQUESTED** [review]

**Comment by mash2k3** [comment]
I believe i updated all your suggestions, let me know if im missing anything?

File: custom_components/bhyve/switch.py [hunk 2]
```diff
@@ -396,6 +401,12 @@ def _setup(self, device):
         status = device.get("status", {})
         watering_status = status.get("watering_status")
 
+        next_start_programs = status.get("next_start_programs")
+
+        next_runtime = orbit_time_to_local_time(
+            status.get("next_start_time")
+        ).isoformat()
```
Comment (sebr) [inline custom_components/bhyve/switch.py:408]: `orbit_time_to_local_time` can return `None`, which will throw an exception when calling `isoformat()`. I'll fix this up.

**Review by sebr — APPROVED** [review]

File: custom_components/bhyve/switch.py (hunk 2 above)
Comment (mash2k3) [reply custom_components/bhyve/switch.py]: Awesome thanks

**Pull request merged**